components/navigation/DocsNav.js

```diff
@@ -19,28 +24,35 @@ const serializedBuckets = buckets.reduce((acc, bucket) => {
 export default function DocsNav({
   item,
   active,
-  onClick = () => {},
+  onClick = () => { },
 }) {
   const subCategories = item.children;
   const bucket = serializedBuckets[item.item.rootSectionId];
 
+  const [openSubCategory, setOpenSubCategory] = useState(active.startsWith(item.item.slug));
+
+  const onClickHandler = () => {
+    setOpenSubCategory(!openSubCategory);
+    onClick();
```

Comment:
Why are adding this here??

Reply:
Because we need to call the functions inherited by the parent component, when clicked. So, it's actually used in the mobile view. Whenever an option is clicked, the navMenu in mobile view gets closed.

Reply:
okay got it